### DISCUSSION TITLE
Open/Zoom plots in new window

### DISCUSSION THREAD

**Discussion by vinaychuri** [posit-dev/positron#4608] [Q&A]
Rstudio has option to open the plots in a new window. The plots could be generated inline within rmarkdown chunk or within the plot pane.

The plot is then opened in a new window for user to sanity check the plot before taking screenshot or right click->Save image

Is there anything equivalent in Positron?

**Comment by juliasilge** [comment]
This is tracked in https://github.com/posit-dev/positron/issues/2270, if you would like to subscribe for updates!
